region/tourian/main.json

```diff
@@ -4227,6 +4227,23 @@
                   ],
                   "note": "IBJ or spring ball bomb jump to avoid the wall jump. Using a total of 3 PBs will allow Samus to roll through the bottom 4 pirates."
                 },
+                {
+                  "name": "G-Mode Morph Pirate Tank",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
```

Comment:
This would require immobile for health which would make this indirect.  Maybe the `enemyDamage` would ensure that is how it plays out?
And a CF would try to kill the 4 pirates with pbs first.  Unless you need the health for future rooms 🤷‍♂️

Reply:
Yes, I think the logic should force it to be indirect. CF would force Samus to stand up, so that wouldn't work.